### PR TITLE
chore: update block capacity

### PR DESCRIPTION
**Pull request by jjyr**
Increase the capacity of a block.

A block can contain 3875 typical 2 in 2 out txs.

### CODE REVIEW

File: resource/specs/testnet.toml [hunk 1]
```diff
@@ -75,7 +75,6 @@ lock.hash_type = "type"
 [params]
 epoch_reward = 1_250_000_00000000
 secondary_epoch_reward = 600_000_00000000
-max_block_cycles = 10_000_000_000
```
Comment (doitian) [inline resource/specs/testnet.toml]: Leave a comment here, that testnet uses the default `MAX_BLOCK_CYCLES` in spec/src/consensus.rs

File: spec/src/consensus.rs [hunk 2]
```diff
@@ -46,8 +48,8 @@ pub(crate) const MIN_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MAX_BLOCK_INTER
 // can divide DEFAULT_EPOCH_REWARD and can be divided by ORPHAN_RATE_TARGET_RECIP.
 pub(crate) const GENESIS_EPOCH_LENGTH: u64 = 1_000;
 
-pub(crate) const MAX_BLOCK_BYTES: u64 = 2_000_000; // 2mb
-pub(crate) const MAX_BLOCK_CYCLES: u64 = TWO_IN_TWO_OUT_CYCLES * 200 * 8;
+pub(crate) const MAX_BLOCK_BYTES: u64 = TWO_IN_TWO_OUT_BYTES * TWO_IN_TWO_OUT_COUNT;
+pub(crate) const MAX_BLOCK_CYCLES: u64 = TWO_IN_TWO_OUT_CYCLES * TWO_IN_TWO_OUT_COUNT;
```
Comment (doitian) [inline spec/src/consensus.rs]: Please go through these constants, make them private if possible.

File: spec/src/lib.rs [hunk 3]
```diff
@@ -52,6 +52,7 @@ pub struct ChainSpec {
 pub struct Params {
     pub epoch_reward: Capacity,
     pub secondary_epoch_reward: Capacity,
+    #[serde(default)]
```
Comment (quake) [inline spec/src/lib.rs]: Maybe not related to this PR, I think that we should move `default` to struct `Params` level, testnet/mainnet use the default value including `max_block_size`, other specs ( dev / integration) can use the different configurable values.

Comment (jjyr) [reply spec/src/lib.rs]: I think maybe we should just remove `params` fields from `testnet.toml`, since actually we do not use this config. @doitian

Comment (jjyr) [reply spec/src/lib.rs]: Apply the change, also remove the `params` from `testnet.toml`.

File: resource/specs/testnet.toml (hunk 1 above)
Comment (jjyr) [reply resource/specs/testnet.toml]: I think @quake 's suggestion is good, and we should remove the entire `params` section from `testnet.toml` until we actually need to config these fields with non-default values.

File: test/src/specs/tx_pool/send_secp_tx.rs [hunk 4]
```diff
@@ -9,13 +9,16 @@ use ckb_resource::CODE_HASH_SECP256K1_BLAKE160_SIGHASH_ALL;
...
-    packed::{CellDep, CellInput, CellOutput, OutPoint, Script},
+    core::{capacity_bytes, Capacity, Cycle, DepType, ScriptHashType, TransactionBuilder},
+    packed::{CellDep, CellInput, CellOutput, OutPoint, Script, Witness},
     prelude::*,
     H256,
 };
 use log::info;
 
+const TX_2_IN_2_OUT_SIZE: usize = 673;
+const TX_2_IN_2_OUT_CYCLES: Cycle = 13_290_092;
```
Comment (TheWaWaR) [inline test/src/specs/tx_pool/send_secp_tx.rs]: Should mention where those values come from.

File: test/src/specs/tx_pool/send_secp_tx.rs [hunk 5]
```diff
@@ -101,34 +104,166 @@ impl Spec for SendSecpTxUseDepGroup {
...
+
+impl Default for CheckTypical2In2OutTx {
+    fn default() -> Self {
+        Self::new(42)
+    }
+}
+
+impl CheckTypical2In2OutTx {
+    pub fn new(seed: u64) -> Self {
+        let mut generator = Generator::non_crypto_safe_prng(seed);
```
Comment (TheWaWaR) [inline test/src/specs/tx_pool/send_secp_tx.rs]: Why certainty is needed for private key?

**Pull request merged**